tests/install_swiftpm_dependencies/project/Makefile

```diff
@@ -0,0 +1,7 @@
+.DEFAULT_GOAL := generate
+
+generate:
+	xcodegen generate
+
+update_tracked_package_resolved:
+	xcodebuild -resolvePackageDependencies -project Demo.xcodeproj
```

Comment:
Now that we decided to `cp package_resolved_fixtures/valid.resolved` after `xcodegen generate`-ing the project, I'm guessing this is not needed anymore (and that one should use `make regenerate_fixtures` from the `tests/install_swiftpm_dependencies/Makefile` instead) and we should thus probably remove that one, right?

```suggestion
```

Reply:
Correct. I clearly missed this cleanup. Thanks!